static/js/rendered_markdown.js

```diff
@@ -164,11 +164,14 @@ export const update_elements = (content) => {
         if (isValid(timestamp)) {
             const text = $(this).text();
             const rendered_time = timerender.render_markdown_timestamp(timestamp, text);
+            const rendered_text =
+                rendered_time.text +
+                ` ${timerender.get_abbreviated_timezone(page_params.timezone)}`;
```

Comment:
I don’t think it’s right to use `page_params.timezone` here. That’s the timezone from the user’s Zulip settings, which may not match the browsers timezone which we use for display.

Reply:
Oh yeah!, Added a workaround to stick only with `Intl.DateTimeFormat().resolvedOptions().timeZone` aka the browser's timezone.